src/Bundle/DependencyInjection/Compiler/RegisterFiltersPass.php

```diff
@@ -28,10 +30,21 @@ public function process(ContainerBuilder $container): void
             return;
         }
 
-        $registry = $container->getDefinition('sylius.registry.grid_filter');
+        $filterRegistry = $container->getDefinition('sylius.registry.grid_filter');
```

Comment:
If `registry` is renamed to `filterRegistry`, then `formTypeRegistry` should be become `filterFormTypeRegister`, hence I propose to remove this change.

```suggestion
        $registry = $container->getDefinition('sylius.registry.grid_filter');
```

Reply:
I like the change